src/main/java/ann/commands/FindCommand.java

```diff
@@ -0,0 +1,14 @@
+package ann.commands;
+
+public class FindCommand extends Command{
+    private String findKeyWords;
```

Comment:
```suggestion
    private String keywords;
```

Do you think it would be more appropriate if this variable was renamed as above? Since 'find' is a verb which can suggest a method style of naming. This could be confusing such as in the `executeCommand` below where you pass in `this.findKeyWords` as an argument, which may seem like a method call.

Reply:
Thanks for pointing this out! I'll fix it 😄